src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -753,6 +761,86 @@ protected Class<?> findClass(String moduleName, String name) {
         return null;
     }
 
+    // The threadLoadingClassMap is a concurrent hashtable that keeps track of the threads
+    // that are currently loading each class by this class loader.  The class name is mapped to
+    // the thread that is first seen loading that class, then removed when loading is complete
+    // for that class or throws an exception.
+    private final Thread curThreadLoadingClass(String name, Thread thread) {
```

Comment:
Suggestion: s/curThreadLoadingClass/threadLoadingClass as the notion of "cur"rent is somewhat ambiguous in this context